ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  CreateTaskView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-02.
+//
+
+import Foundation
+import SwiftUI
+
+struct CreateTaskView: View {
+    @StateObject private var createTaskViewModel = CreateTaskViewModel()
+    @State var titleInput: String = ""
+    @State var showDatePicker = false
+    @State var chooseDateTitle: String = "Choose date"
+    @State var date: Date = Date()
+    var body: some View {
+        NavigationStack {
+            VStack(alignment: .leading, spacing: 15) {
+                Text("Task Title")
+                    .multilineTextAlignment(.leading)
+                TextField("Insert Title", text: $titleInput)
+                    .padding(.horizontal, 10.0)
+                    .textInputAutocapitalization(.words)
+                    .frame(maxWidth: 390, maxHeight: 50)
```

Comment:
no width should be specified
```suggestion
                    .frame(maxHeight: 50)
```
also I messed up min and max during previous review cycle. it should be min
```suggestion
                    .frame(minHeight: 44)
```
please also use sizes from Figma I noticed that the text field height should be 44 for both title and date fields